atomic_reactor/utils/cachito.py

```diff
@@ -82,7 +83,7 @@ def request_sources(self, repo, ref, flags=None, pkg_managers=None, user=None,
         # Remove None values
         payload = {k: v for k, v in payload.items() if v}
 
-        url = '{}/api/v1/requests'.format(self.api_url)
+        url = self.api_endpoint
         logger.debug('Making request %s with payload:\n%s', url, json.dumps(payload, indent=4))
         response = self.session.post(url, json=payload)
```

Comment:
In this case, can we just use the self.api_endpoint in the post call (and in the log msg) and get rid of the attribution to url?